client/src/utils/BGraphQueryUtil.ts

```diff
@@ -148,3 +149,132 @@ export function bioNodeOutDegree (bgraphPathQuery, clause: Filter): any {
     return false;
   });
 }
+
+// EPI PATH QUERIES
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function epiPathPre (bgraphPathQuery, clause: Filter): any {
+  return bgraphPathQuery.filter(document => {
+    if (document._type === 'node') {
+      const node = document;
+      return clause.values[0] === node.label;
+    }
+    // Document type is an edge
+    return false;
+  })
+    .start()
+    .unique();
+}
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/explicit-module-boundary-types
+export function epiPathPreNodeFilterLoop (bgraphPathQuery, _clause: Filter): any {
+  return bgraphPathQuery.in(); // TODO: Why are edge directions backward??
+}
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/explicit-module-boundary-types
+export function epiPathPreEdgeFilterLoop (bgraphPathQuery, _clause: Filter): any {
```

Comment:
I think we use arrow functions for our methods across the utils files so we probably want to be consistent here as well.